src/lib/Scenes/Partner/Partner.tsx

```diff
@@ -0,0 +1,176 @@
+import { Box, Button, Flex, Serif, Spacer, Theme } from "@artsy/palette"
+import { Partner_partner } from "__generated__/Partner_partner.graphql"
+import { PartnerFollowMutation } from "__generated__/PartnerFollowMutation.graphql"
+import { PartnerQuery } from "__generated__/PartnerQuery.graphql"
+import { RetryErrorBoundary } from "lib/Components/RetryErrorBoundary"
+import { ScrollableTab } from "lib/Components/ScrollableTabBar"
+import TabBar from "lib/Components/TabBar"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { SafeAreaInsets } from "lib/types/SafeAreaInsets"
+import { get } from "lib/utils/get"
+import renderWithLoadProgress from "lib/utils/renderWithLoadProgress"
+import React, { useState } from "react"
+import ScrollableTabView from "react-native-scrollable-tab-view"
+import { commitMutation, createRefetchContainer, graphql, QueryRenderer, RelayRefetchProp } from "react-relay"
+import { PartnerArtworkFragmentContainer as PartnerArtwork } from "./Components/PartnerArtwork"
+import { PartnerOverviewFragmentContainer as PartnerOverview } from "./Components/PartnerOverview"
+import { PartnerShowsFragmentContainer as PartnerShows } from "./Components/PartnerShows"
+
+const INITIAL_TAB = 0
+
+export const Partner: React.FC<{
+  partner: Partner_partner
+  relay: RelayRefetchProp
+}> = ({ partner, relay }) => {
+  const [selectedTab, setSelectedTab] = useState(INITIAL_TAB)
+  const [isFollowedChanging, setIsFollowedChanging] = useState(false)
+  console.log("tab", selectedTab)
+
+  const handleFollowPartner = () => {
+    const {
+      slug: partnerSlug,
+      profile: { isFollowed: partnerFollowed, internalID: profileID },
+    } = partner
+
+    setIsFollowedChanging(true)
+    commitMutation<PartnerFollowMutation>(relay.environment, {
+      onCompleted: () => setIsFollowedChanging(false),
+      onError: e => console.log("errors", e),
+      mutation: graphql`
+        mutation PartnerFollowMutation($input: FollowProfileInput!) {
+          followProfile(input: $input) {
+            profile {
+              id
+              slug
+              internalID
+              isFollowed
+            }
+          }
+        }
+      `,
+      variables: {
+        input: {
+          profileID,
+          unfollow: partnerFollowed,
+        },
+      },
+      optimisticResponse: {
+        followProfile: {
+          profile: {
+            id: partner.profile.id,
+            internalID: profileID,
+            slug: partnerSlug,
+            isFollowed: !partnerFollowed,
+          },
+        },
+      },
+    })
+  }
+
+  const isFollowed = get(partner, p => p.profile.isFollowed)
+
+  return (
+    <Theme>
+      <Flex style={{ flex: 1 }}>
+        <Spacer mt={60} />
+        <Flex flexDirection="row" flexWrap="nowrap" justifyContent="space-between">
+          <Box px={2} pb={2}>
+            <Serif size="6">{partner.name}</Serif>
+          </Box>
+          {isFollowed && (
```

Comment:
Seems like this `Button` won't get rendered if the partner isn't being followed?